client/app/veeam-enterprise/veeam-enterprise.js

```diff
@@ -0,0 +1,22 @@
+angular.module('managerApp').config(($stateProvider) => {
+  $stateProvider
+    .state('paas.veeam-enterprise', {
+      abstract: true,
+      url: '/veeam-enterprise/{serviceName}',
+      templateUrl: 'app/veeam-enterprise/veeam-enterprise.html',
+      controller: 'VeeamEnterpriseCtrl',
+      controllerAs: 'VeeamEnterpriseCtrl',
```

Comment:
Why not prefer use `$ctrl` in that case?

Reply:
Could have used a Component instead

Reply:
To keep the same way as the others pages

Reply:
If you pay attention, you also add a state `paas.veeam-enterprise.dashboard` (L11) where you set `$ctrl` as `controllerAs`.

In that way we definitely tend to homogenize our codebase.

Do you feel agree with that?

Reply:
In fact, i've almost kept the same way as `veeam` folder. But I agree we could go with default `$ctrl` for this 2 pages :)